lieer/gmailieer.py

```diff
@@ -29,6 +29,8 @@ def main (self):
     common.add_argument ('-s', '--no-progress', action = 'store_true',
         default = False, help = 'Disable progressbar (always off when output is not TTY)')
 
+    common.add_argument ('account', type = str, help = 'GMail account to use')
```

Comment:
Hmm, would it be feasible to allow an empty value here mean "all accounts"?

Reply:
I'm not sure that's so elegant or intuitive. Perhaps we should have a default account in the global section of the config, similar to what mbsync does?

Reply:
It's probably best you leave this as-is, then. I was just considering the home-manager service module, which needs to set up a systemd timer per-account. However, it's best that way, since you could configure options and timer intervals per-account.